packages/dev/core/src/PostProcesses/RenderPipeline/Pipelines/lensRenderingPipeline.ts

```diff
@@ -574,24 +576,18 @@ export class LensRenderingPipeline extends PostProcessRenderPipeline {
     private _createGrainTexture(): void {
         const size = 512;
 
-        this._grainTexture = new DynamicTexture("LensNoiseTexture", size, this._scene, false, Texture.BILINEAR_SAMPLINGMODE);
-        this._grainTexture.wrapU = Texture.WRAP_ADDRESSMODE;
-        this._grainTexture.wrapV = Texture.WRAP_ADDRESSMODE;
-
-        const context = (<DynamicTexture>this._grainTexture).getContext();
-
-        const rand = (min: number, max: number) => {
-            return Math.random() * (max - min) + min;
-        };
-
-        let value;
-        for (let x = 0; x < size; x++) {
-            for (let y = 0; y < size; y++) {
-                value = Math.floor(rand(0.42, 0.58) * 255);
-                context.fillStyle = "rgb(" + value + ", " + value + ", " + value + ")";
-                context.fillRect(x, y, 1, 1);
-            }
+        const data = new Uint8Array(size * size * 3);
+        for (let index = 0; index < data.length; ) {
+            const value = Math.floor(Scalar.RandomRange(0.42, 0.58) * 255);
+            data[index++] = value;
+            data[index++] = value;
+            data[index++] = value;
         }
-        (<DynamicTexture>this._grainTexture).update(false);
+
+        const texture = RawTexture.CreateRGBTexture(data, size, size, this._scene, false, false, Constants.TEXTURE_BILINEAR_SAMPLINGMODE);

```

Comment:
RGB textures are not supported in WebGPU, you should create a RGBA instead.

Reply:
fixed